### PR TITLE
Feature/separate tokens

### PR DESCRIPTION
**Pull request by MakMuftic**

### CODE REVIEW

File: internal/hactar/hactar.go [hunk 1]
```diff
@@ -90,13 +87,17 @@ func (c *Client) Auth(email string, password string) (string, error) {
 		return "", errors.New(fmt.Sprintf("Unable to authorize, server returned http status %s", response.Status))
 	}
 
-	return tokenResponse.Token, err
+	buf := new(bytes.Buffer)
+	_, _ = buf.ReadFrom(response.Body)
+	token := buf.String()
```
Comment (morrigan) [inline internal/hactar/hactar.go]: Just curious, why did you change it to use buffers?

**Review by morrigan — APPROVED** [review]
Looks good!

File: internal/hactar/hactar.go (hunk 1 above)
Comment (MakMuftic) [reply internal/hactar/hactar.go]: Because the response from hactar-backend is not json file `{token: "test-toke"}` as I expected but plain text inside the body, so I can't parse it to structure but need to read it this way.

Comment (morrigan) [reply internal/hactar/hactar.go]: Umm, it is possible that's a mistake? You should get JSON type content, same as is in communication with lotus. Should have raised an issue I guess

Comment (MakMuftic) [reply internal/hactar/hactar.go]: I agree, will raise an issue on hactar-backend and change code in this PR.
